/**:
    ros__parameters:
        camera_name: "jetcocam"
        frame_id: "jetcocam"
        camera_info_url: "package://jetcobot_bringup/config/jetcocam_1_calib.yaml"
        publish_rate: 1.0
